### DISCUSSION TITLE
Syncing PrimeOne Token Studio with Azure DevOps (ADO)

### DISCUSSION THREAD

**Discussion by tophersteven** [primefaces/community#1861] [Figma UI Kit]
Has anyone had success syncing the PrimeOne Library using Token Studio with Azure DevOps? I get a “check credentials” error but our systems guy is telling me that I should have full access.

**Comment by onursenture** [comment]
We do not have information about PrimeOne's use of Azure DevOps. However, if full access is needed, perhaps you did not set the necessary permissions in the PrimeOne file you imported into Figma. If you want more detailed support on the issue, please ask the [Figma Forum](https://forum.figma.com/) or a support team related to Azure DevOps.
